src/wp-includes/option.php

```diff
@@ -2284,6 +2294,16 @@ function delete_network_option( $network_id, $option ) {
 		 */
 		do_action( 'delete_site_option', $option, $network_id );
 
+		$notoptions_key = "$network_id:notoptions";
+		$notoptions = wp_cache_get( $notoptions_key, 'site-options' ); // Yes, again... we need it to be fresh.
```

Comment:
CS: Alignment. Cache is not queried earlier in the function.

```suggestion
		$notoptions_key = "$network_id:notoptions";
		$notoptions     = wp_cache_get( $notoptions_key, 'site-options' );
```